specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -247,51 +195,90 @@ model ImageCategoriesAnalysis {
   severity?: int32;
 }
 
-@added(ContentSafety.Versions.v2024_01_30_Preview)
+@added(ContentSafety.Versions.v2023_10_30_Preview)
 @doc("The image batch analysis request.")
 model BatchAnalyzeImagesOptions {
-  @doc("The URL of the Azure Storage Blob containing all the images to be analyzed in the batch task.")
-  imagesBlobFolderUrl: url;
+  @doc("The blob parameters of images to be analyzed.")
+  images: BatchAnalyzeImages;
 
-  @doc("The URL of the Azure Storage Blob where the batch task results will be written.")
-  analysisResultsBlobFolderUrl: url;
+  @doc("The blob parameters of result files.")
+  analysisResults: BatchAnalyzeImagesResults;
```

Comment:
This is a very confusing name. Even I had to review this several times over again. You might consider not putting anything called "results" in a request model, or at least something like `saveResultsLocation`.

Reply:
I changed the name to "resultsBlob" as ChatGPT suggested. I did not use the word "location" because there is one "blobPrefixLocation" under this concept.